atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -498,6 +499,22 @@ def get_koji_upload_dir():
         unique_fragment = '%r.%s' % (time.time(), random_chars)
         return os.path.join(dir_prefix, unique_fragment)
 
+    def _update_content_versions(self, worker_reactor_conf):
+        enable_v1 = False
+        for descriptor in self.platform_descriptors:
+            if descriptor['platform'] == platform:
+                enable_v1 = descriptor.get('enable_v1', False)
+                break
+
+        # Remove v1 from content_versions if the platform descriptor doesn't enable v1
+        if not enable_v1 and 'content_versions' in worker_reactor_conf:
+            try:
+                worker_reactor_conf['content_versions'].remove('v1')
+            except ValueError:
+                pass
+            if not worker_reactor_conf['content_versions']:
+                raise RuntimeError("v1 not enabled in platform descriptor for v1-only platform")
```

Comment:
Need to test the new code paths here.

Reply:
yop I will work on unit tests :)

Reply:
This error message doesn't seem right to me. What's a "v1-only" platform? You probably mean that there's no v1 enabled, and this is a v1-only build request.

Would it be better to log up above that `'v1'` is being removed for this platform, then here we can simply say that `content_versions` is empty.

Reply:
ok will change that